docs/core/rules.rst

```diff
@@ -0,0 +1,187 @@
+:desc: Use Rasa Open Source rules to respond to FAQs, fill forms, or handle
+       fallbacks gracefully.
+
+.. _rules:
+
+Rules
+=====
+
+.. edit-link::
+
+.. contents::
+   :local:
+
+Rasa rules are a form of training data used to train Rasa's dialogue management models.
+Rules describe parts of conversations which should always follow the same path.
+Useful applications for rules are for example:
+
+* **FAQs**: FAQs are questions which users ask independent of the current context.
+  Rules are an easy way to specify fixed answers to these questions.
+
+* :ref:`fallback-actions`: Users might confront your assistant with unexpected messages.
+  The messages will typically receive low intent confidence.
+  You can use the :ref:`fallback-classifier` in combination with rules to treat
+  messages with low NLU confidence like an FAQ.
+
+* :ref:`forms`: It's a common use case for assistants to collect form-like
+  data from the user. Both, activation of forms as well as handling of unexpected
+  events as part of a form, are often following fixed paths.
+
+**Don't overuse rules**. They are great to handle small specific conversation parts but
+unlike :ref:`stories` rules don't have the power to generalize to unseen conversation
+paths. Combine rules and stories to make your assistant robust and able to handle
+real user behavior.
+
+Writing a Rule
+--------------
+
+Before you start writing rules, you have to make sure that the :ref:`rule-policy`
+is added to your model configuration:
+
+.. code-block:: yaml
+
+    policies:
+    - ... # Other policies
+    - name: RulePolicy
+
+Rules can then be added to the ``rules`` section of your training data.
+
+Rules for the Conversation Start
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To write a rule which only applies at the beginning of a conversation, start with the
+intent which starts the conversation and then add the actions which your assistant
+should perform in response to that.
+
+.. code-block:: yaml
+
+    rules:
+
+    - rule: Say `hello` when the user starts a conversation with intent `greet`
+      steps:
+      - intent: greet
+      - action: utter_greet
+
+.. _rules-applying-anytime:
+
+Rules which Apply Anytime
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To indicate that a rule can apply at any point in a conversation, specify ``...`` as
+first conversation step. The ``...`` indicate that the rule applies independent of the
+previous conversation.
```

Comment:
this will completely change. Would it make sense to write the docs for the new version?

Reply:
see my comment here https://github.com/RasaHQ/rasa/pull/6195#issuecomment-662407289